source/computer/fedora-setup.rst

```diff
@@ -176,7 +171,7 @@ Intel 软件开发工具包
 ^^^^^^^^^^^^^^^^^^^^
 
 `Intel oneAPI Toolkits <https://software.intel.com/content/www/us/en/develop/tools/oneapi.html>`__
-是英特尔最新的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc`` 和 ``ifort`` 命令）。
+是 Intel 公司开发的软件开发工具包。它也提供了 C/C++ 编译器和 Fortran 编译器（``icc``\，``i++`` 和 ``ifort`` 命令）。
```

Comment:
Please check if it provides the `i++` command as a C++ compiler.

Reply:
no, see https://github.com/seismo-learn/seismology101/pull/296#discussion_r614501547.